strawberryfields/ops.py

```diff
@@ -625,28 +625,29 @@ def __init__(self, alpha=0, p=0):
         super().__init__([alpha, p])
 
     def _apply(self, reg, backend, **kwargs):
-        p = par_evaluate(self.p)
-        alpha = p[0]
-        phi = np.pi * p[1]
+        alpha = self.p[0]
+        phi = np.pi * self.p[1]
         D = backend.get_cutoff_dim()
         l = np.arange(D)[:, np.newaxis]
 
         # normalization constant
-        temp = np.exp(-0.5 * np.abs(alpha)**2)
-        N = temp / np.sqrt(2*(1 + np.cos(phi) * temp**4))
+        temp = pf.exp(-0.5 * pf.Abs(alpha)**2)
```

Comment:
Is it really `sympy.Abs`? Weird :thinking:

Reply:
It's to make sure it's not mixed up with the Python built-in function `abs`. It's the same thing with `min` and `max`.